typescript/libs/view_helpers.py

```diff
@@ -106,8 +106,9 @@ def is_typescript(view):
     except:
         return False
 
-    return (view.match_selector(location, 'source.ts') or
-            view.match_selector(location, 'source.tsx'))
+    is_ts_file = view.match_selector(location, 'source.ts') or view.match_selector(location, 'source.tsx')
+    is_js_file = view.match_selector(location, 'source.jsx') and cli.enable_language_service_for_js
```

Comment:
Does Sublime consider the entire region of a .js file to be a `source.jsx` scope? Or is this from supplying our own `tmLanguage`? Can you add a comment explaining, since it's not obvious why you're not checking for `scope.js`.

Reply:
I am using `view.match_selector(location, 'source.ts, source.tsx, source.js, source.jsx')` here and it works perfectly.